docs/NEW_RULES.md

```diff
@@ -1,5 +1,4 @@
 # How to add a new validation rule _i.e._ a validator?
-
 Adding a new validator is relatively simple:
```

Comment:
IMHO this documentation makes the process for adding new rules seem way more complicated than it actually is. I think the problem is that it's currently written with a lot of technical jargon that will be intimidating to a lot of people.

I think this needs to be written more as a narrative walking through an example (think a blog post) than an abstract bulleted list and in plainer English understandable to the layperson. See https://github.com/CUTR-at-USF/gtfs-realtime-validator/blob/master/ADDING_NEW_RULES.md for an example (although that's really too long also and could be simplified).